packages/compass-indexes/src/components/search-indexes-modals/create-search-index-modal.tsx

```diff
@@ -103,6 +103,9 @@ export const CreateSearchIndexModal: React.FunctionComponent<
     (track) => {
       if (isModalOpen) {
         track('Screen', { name: 'create_search_index_modal' });
+        track('Index Create Opened', {
```

Comment:
Should we ask if we can use the `Screen` event with `create_search_index_modal` instead? It seems like we're duplicating the call here when we could filter for the same event in our telemetry. Fine as is too, this is a nit on reducing duplication, feel free to merge as is imo.